src/trustme/__init__.py

```diff
@@ -146,11 +146,11 @@ def bytes(self) -> bytes:
         """
         return self._data
 
-    def write_to_path(self, path: str, append: bool = False) -> None:
+    def write_to_path(self, path: Union[str, "os.PathLike[str]"], append: bool = False) -> None:
         """Writes the data to the file at the given path.
 
         Args:
-          path (str): The path to write to.
+          path (str or path-like object): The path to write to.
```

Comment:
This breaks documentation, should we use `os.PathLike[str]` here or find a way to ignore the Sphinx failure?

Reply:
Fixed, thanks. BTW, these docstring types are mostly redundant with the function typing, I can [configure sphinx](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_typehints) accordingly and remove them if you agree.

Reply:
I agree.